### ISSUE TITLE
Use this to proxify TCP (non HTTP) traffic?

### ISSUE THREAD

**Issue by julien-c** [joeferner/node-http-mitm-proxy#158]
My network knowledge is a bit rusty, but I'd like to know if anyone had any success proxifying other (SSL-encrypted) TCP protocols than HTTP?

Feel free to close if not relevant to this particular project.

**Comment by GregRos** [comment]
Not a developer, but I can guarantee from the source code that this is an HTTP or Web proxy. They are very different from a lower-level proxy.

An HTTP proxy is essentially an HTTP server that responds to HTTP requests of a certain form. It doesn't handle TCP traffic.

**Comment by julien-c** [comment]
I would tend to object to your argument that this lib also proxifies websockets (which although using HTTP for connection are pretty much a whole different TCP protocol afterwards).

**Comment by GregRos** [comment]
@julien-c Yes, that is correct. In addition to creating a server for handling HTTP requests, it also creates a server for handling WS requests. However, in both cases, these are just web servers that accept high-level requests (HTTP/WS) and move them forward using the same protocol. 

Again, in the source code itself, which isn't very long, you can see there is no lower-level stuff going on.
